.grit/patterns/openai.md

```diff
@@ -23,11 +23,44 @@ pattern rename_resource() {
         `Image` => `images`,
         `Model` => `models`,
         `Moderation` => `moderations`,
-        // TODO: `Customer`, `Deployment`, `Engine`, `ErrorObject` are not converted to the new version.
-        //       There seems to be no equivalent items in the new version
     }
 }
 
+pattern rename_resource_cls() {
+    or {
+        r"Audio" => `resources.Audio`,
+        r"ChatCompletion" => `resources.chat.Completions`,
+        r"Completion" => `resources.Completions`,
+        r"Edit" => `resources.Edits`,
+        r"Embedding" => `resources.Embeddings`,
+        r"File" => `resources.Files`,
+        r"FineTune" => `resources.FineTunes`,
+        r"FineTuningJob" => `resources.FineTuning`,
+        r"Image" => `resources.Images`,
+        r"Model" => `resources.Models`,
+        r"Moderation" => `resources.Moderations`,
+    }
+}
+
+pattern depreciated_resource() {
```

Comment:
```suggestion
pattern deprecated_resource() {
```

Reply:
I knew the spelling felt suspicious. I have the same error in other places.